terraform/services/snyk-integration/main.tf

```diff
@@ -1,15 +1,19 @@
-data "aws_iam_policy" "developer_boundary_policy" {
-  name = "developer-boundary-policy"
-}
-
 data "aws_ssm_parameter" "external_id" {
-  name = "/snyk-integration/external-id"
+  name = local.external_id_param
```

Comment:
Isn't the `/snyk-integration/external-id` param being pulled in each account? I think we could leave the terraform as is and update the params in the AB2D accounts.

Reply:
ok let me revert terraform

Reply:
I will just close this pull request and update the param in ab2d